src/middlewared/middlewared/etc_files/local/users.oath

```diff
@@ -0,0 +1,6 @@
+<%
+	twofactor_auth = middleware.call_sync('auth.twofactor.config')
+%>\
+% for user in middleware.call_sync('user.query'):
+${f'HOTP/T{twofactor_auth["interval"]}/{twofactor_auth["otp_digits"]}'}	${user['username']}	-	${twofactor_auth['secret_hex']}
```

Comment:
So all users will have the same two factor? I wonder if we shouldnt restrict this only for uid 0